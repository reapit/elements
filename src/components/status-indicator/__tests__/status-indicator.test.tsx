import { render } from '@testing-library/react'
import { StatusIndicator } from '..'

describe('StatusIndicator', () => {
  it('should render as expected', () => {
    expect(render(<StatusIndicator variant="neutral">Neutral</StatusIndicator>).asFragment()).toMatchSnapshot()
    expect(render(<StatusIndicator variant="success">Success</StatusIndicator>).asFragment()).toMatchSnapshot()
    expect(render(<StatusIndicator variant="warning">Warning</StatusIndicator>).asFragment()).toMatchSnapshot()
    expect(render(<StatusIndicator variant="pending">Pending</StatusIndicator>).asFragment()).toMatchSnapshot()
    expect(render(<StatusIndicator variant="danger">Danger</StatusIndicator>).asFragment()).toMatchSnapshot()
    expect(render(<StatusIndicator variant="inactive">Inactive</StatusIndicator>).asFragment()).toMatchSnapshot()
    expect(render(<StatusIndicator variant="accent1">Accent 1</StatusIndicator>).asFragment()).toMatchSnapshot()
    expect(render(<StatusIndicator variant="accent2">Accent 2</StatusIndicator>).asFragment()).toMatchSnapshot()
  })
})
