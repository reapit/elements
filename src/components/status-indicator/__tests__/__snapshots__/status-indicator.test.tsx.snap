// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`StatusIndicator component > should match a snapshot 1`] = `
<DocumentFragment>
  <span
    class="el-intent-primary el-status-indicator"
  />
</DocumentFragment>
`;

exports[`StatusIndicator component > should match a snapshot 2`] = `
<DocumentFragment>
  <span
    class="el-intent-primary el-status-indicator"
  />
</DocumentFragment>
`;

exports[`StatusIndicator component > should match a snapshot 3`] = `
<DocumentFragment>
  <span
    class="el-intent-pending el-status-indicator"
  />
</DocumentFragment>
`;

exports[`StatusIndicator component > should match a snapshot 4`] = `
<DocumentFragment>
  <span
    class="el-intent-danger el-status-indicator"
  />
</DocumentFragment>
`;

exports[`StatusIndicator component > should match a snapshot 5`] = `
<DocumentFragment>
  <span
    class="el-intent-success el-status-indicator"
  />
</DocumentFragment>
`;

exports[`StatusIndicator component > should match a snapshot 6`] = `
<DocumentFragment>
  <span
    class="el-intent-warning el-status-indicator"
  />
</DocumentFragment>
`;

exports[`StatusIndicator component > should match a snapshot 7`] = `
<DocumentFragment>
  <span
    class="el-intent-neutral el-status-indicator"
  />
</DocumentFragment>
`;

exports[`StatusIndicator component > should match a snapshot 8`] = `
<DocumentFragment>
  <span
    class="el-intent-default el-status-indicator"
  />
</DocumentFragment>
`;
